bucket/flutter.json

```
@@ -0,0 +1,46 @@
+{
+    "homepage": "https://flutter.dev/",
+    "description": "Google’s mobile app SDK for crafting high-quality native interfaces on iOS and Android in record time. ",
+    "version": "1.5.4-hotfix.2",
+    "license": "BSD-3-Clause",
+    "url": [
+        "https://storage.googleapis.com/flutter_infra/releases/stable/windows/flutter_windows_v1.5.4-hotfix.2-stable.zip",
+        "https://raw.githubusercontent.com/ScoopInstaller/Main/master/scripts/flutter-dev-setup.ps1"
+    ],
+    "hash": [
+        "1d5e1779430be6bbb4fa49f4bccc7c5d24a76f3d2a100129e09b4b8eef33b1b4",
+        "bbd8dd269dd70d97e0224025281e55b7e2e32364d5c47e082ca7f45e33d1a613"
+    ],
+    "depends": [
+        "android-sdk",
+        "java/adopt8-hotspot"
+    ],
+    "extract_dir": "flutter",
+    "bin": [
+        "bin\\flutter.bat",
+        "flutter-dev-setup.ps1"
+    ],
+    "env_add_path": "bin\\cache\\dart-sdk",
+    "post_install": [
+        "flutter-dev-setup.ps1",
```

If you want to execute this I doubt it will work as context of execution will be set to Execution path.

```suggestion
        "& \"$dir\\flutter-dev-setup.ps1\"",
```